ModularTegustation/tegu_items/debug_items.dm

```diff
@@ -374,3 +374,26 @@
 	say("Restarting...")
 	SLEEP_CHECK_DEATH(10)
 	DeepsCheckStart()
+
+//breach tester
+/obj/item/breachtester//for testing many abnormalities very quickly
+	name = "Breach tester"
+	desc = "For testing use only, DO NOT DISTRIBUTE! Breach types can be checked under _DEFINES/abnormalities.dm"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "nanoimplant"
+	var/breach_type = BREACH_NORMAL
+	var/list/breach_list = list(
+			BREACH_NORMAL, BREACH_PINK, BREACH_MINING,
```

Comment:
why have a multi-line list, just to put all its contents on the same line?
Bit weird, also lists are supposed to end on the same line they start at, AKA:
```
var/list/something = list(
    object,
)
```
and not
```
var/list/something = list(
    object,
    )